config/routes.rb

```diff
@@ -146,7 +146,6 @@
     resources :response_templates, only: %i[index create edit update destroy]
     resources :feedback_messages, only: %i[index create]
     resources :organizations, only: %i[update create destroy]
-    resources :followed_articles, only: [:index]
```

Comment:
This is not directly related to this PR but I tried to see where `User#followed_articles` is used and stumbled upon this route which actually 404s:

```
ActionController::RoutingError - uninitialized constant FollowedArticlesController
Did you mean?  FollowsController:
```